pkg/setting/setting.go

```diff
@@ -179,6 +179,9 @@ var (
 	GrafanaComUrl string
 
 	ImageUploadProvider string
+
+	// Usage Insights
+	UsageInsightsEnabled bool
```

Comment:
Not ideal, but looked (imho) safer than reading it directly from settings (e.g. Settings Provider) and "duplicating" or "re-defining" its default value. So, I defined it here to let the owner service (MetaAnalytics) to set the corresponding value.